stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -1,13 +1,44 @@
 # Stubs for multiprocessing
 
-from typing import Any, Callable, Iterable, Mapping
+from typing import Any, Callable, Iterable, Mapping, Sequence
 
 from multiprocessing.process import current_process as current_process
 
 class Lock():
     def acquire(self, block: bool = ..., timeout: int = ...) -> None: ...
     def release(self) -> None: ...
 
+class AsyncResult():
+    def get(self, timeout: float = -1) -> Any: ...
+    def wait(self, timeout: float = -1) -> None: ...
+    def ready(self) -> bool: ...
+    def successful(self) -> bool: ...
+
+class Pool():
+    def __init__(self, processes: int = ...) -> None: ...
```

Comment:
According to the docs this has a lot more arguments. Also, it is documented that `processes = None` has a special meaning, so the type should really be `Optional[int]`.